common/src/app/common/types/tokens_lib.cljc

```diff
@@ -30,7 +30,7 @@
   "Decompose a string in the form 'one.two.three' into a vector of strings, removing spaces."
   [path separator]
   (let [xf (comp (map str/trim)
-                 (remove str/empty?))]
+                 #_(remove str/empty?))]
```

Comment:
This causes that if a path has no groups, the name is put in the first element, and ends up being the "group" name. We need to think a way to avoid this.